src/client/North/NewApplicationRow.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Button, Form, Col, Row } from 'reactstrap'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
just for info.  I have seen a new package called nanoid for generatiing unique ids.

Reply:
Ok, thanks. As i read, it's faster than uuid and has a smaller size with no additional dependencies. I am going to switch the library